ember_debug/utils/type-check.js

```diff
@@ -9,8 +9,16 @@ const { ComputedProperty } = Ember;
  */
 export function isComputed(object, key) {
   // Ember > 3.10
-  if (Ember.Debug.isComputed) {
-    return Ember.Debug.isComputed(object, key) || getDescriptorFor(object, key);
+  if (Ember.Debug.isComputed && Ember.Debug.isComputed(object, key)) {
+    return true;
+  }
+
+  if (Ember.meta(object) && Ember.meta(object).peekDescriptors(key)) {
+    return !!Ember.meta(object).peekDescriptors(key)._getter;
+  }
+
+  if (getDescriptorFor(object, key) instanceof ComputedProperty) {
```

Comment:
@pzuraq will this always be `instanceof ComputedProperty`? I thought there was something where things were not instances of that anymore, but maybe I am remembering wrong.

Reply:
Newer code should never reach here, above it uses the 3.10+ method for checking and anything that's more recent should hit that first.